app/database_etl/owid_ingestion_handler/owid_data_handler.py

```diff
@@ -90,3 +91,8 @@ def get_vaccinated(country_name: str, midpoint_date: datetime, fully_vaccinated=
     col_name = 'people_fully_vaccinated_per_hundred' if fully_vaccinated else 'people_vaccinated_per_hundred'
     ret = country_data.loc[country_data['date'] == target_date][col_name]
     return float(ret) if not ret.empty else None
+
+
+def get_whether_exact_match(country_name: str, estimate_grade: str):
```

Comment:
let's remove this country_name param if unused for now

Reply:
Done.